Command/MigrateConfigCommand.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
+
+use Jose\Bundle\JoseFramework\JoseFrameworkBundle;
+use Jose\Component\Checker\ClaimCheckerManager;
+use Jose\Component\Core\JWK;
+use Jose\Component\Core\JWKSet;
+use Jose\Component\Encryption\JWEBuilder;
+use Jose\Component\Encryption\JWELoader;
+use Jose\Component\KeyManagement\JWKFactory;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\KeyLoader\KeyLoaderInterface;
+use Symfony\Bundle\FrameworkBundle\Command\AbstractConfigCommand;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\Compiler\ValidateEnvPlaceholdersPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\ConfigurationExtensionInterface;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * @author Nicolas Cabot <n.cabot@lexik.fr>
+ *
+ * @final
+ */
+#[AsCommand(name: 'lexik:jwt:migrate-config', description: 'Migrate LexikJWTAuthenticationBundle configuration to the Web-Token one.')]
+class MigrateConfigCommand extends AbstractConfigCommand
+{
+    /**
+     * @deprecated
+     */
+    protected static $defaultName = 'lexik:jwt:migrate-config';
+
+    /**
+     * @var KeyLoaderInterface
+     */
+    private $keyLoader;
+
+    /**
+     * @var string
+     */
+    private $signatureAlgorithm;
+
+    /**
+     * @var string
+     */
+    private $passphrase;
+
+    public function __construct(
+        KeyLoaderInterface $keyLoader,
+        string $passphrase,
+        string $signatureAlgorithm
+    ) {
+        parent::__construct();
+        $this->keyLoader = $keyLoader;
+        $this->passphrase = $passphrase === '' ? null : $passphrase;
+        $this->signatureAlgorithm = $signatureAlgorithm;
+    }
+
+    public function isEnabled(): bool
+    {
+        $requirements = [
+            JoseFrameworkBundle::class => 'web-token/jwt-bundle',
+            JWKFactory::class => 'web-token/jwt-key-mgmt',
+            ClaimCheckerManager::class => 'web-token/jwt-checker',
+            JWEBuilder::class => 'web-token/jwt-encryption',
+            JWELoader::class => 'web-token/jwt-signature',
```

Comment:
probably not the right class as JWELoader is from `web-token/jwt-encryption`

Reply:
Reasons given in https://github.com/lexik/LexikJWTAuthenticationBundle/pull/1041#discussion_r958838652